linkup-cli/src/local_dns.rs

```diff
@@ -0,0 +1,196 @@
+use std::{
+    fs,
+    process::{Command, Stdio},
+};
+
+use crate::{
+    linkup_file_path,
+    local_config::{config_path, get_config},
+    services, CliError, Result, LINKUP_CF_TLS_API_ENV_VAR, LINKUP_LOCALDNS_INSTALL,
+};
+
+pub fn install(config_arg: &Option<String>) -> Result<()> {
```

Comment:
One thing we might consider doing here is making sure we're on macos? I'm not sure what the linux support is like 😅

Reply:
Hmmm, that is a good point. We can take a look it as well!
I guess we can find someone with a Linux and try it out as well 👀

Reply:
It's more of a nice to have, we can take a look at it later!